src/2023/schedule/index.html

```diff
@@ -41,6 +41,54 @@ <h2 class="schedule-item-title">Registration</h2>
             <h2 class="schedule-item-title">Opening words</h2>
```

Comment:
```suggestion
            <h2 class="schedule-item-title">Start of the conference - opening words</h2>
```

Reply:
I'd add this to make it clear that people should be there and seated at 9:30. As "Opening words" can be easily misread as "Opening doors" and people may do this mistake and come just after 9:45.

Reply:
I don't have a strong opinion on this, but I think the schedule item above "Registration" makes it clear enough that "Opening words" is not "Opening doors". 🤷